src/pudl/metadata/fields.py

```diff
@@ -738,6 +742,38 @@
         "description": "Average monthly coincident peak (CP) demand (for requirements purchases, and any transactions involving demand charges). Monthly CP demand is the metered demand during the hour (60-minute integration) in which the supplier's system reaches its monthly peak. In megawatts.",
         "unit": "MW",
     },
+    "company_id_sec": {
+        "type": "string",
+        "description": "Assigned identifier for companies that file SEC10k forms, or show up in Exhibit 21 attachments.",
+    },
```

Comment:
What differentiates the `company_id_sec` from the `central_index_key`?  Are we assigning one of them and the SEC assigning the other? Is there a 1-to-1 mapping between them and they just get used in different contexts? Can we say something about how they relate to each other (or don't) in the field descriptions?

Reply:
The `company_id_sec`  is assigned by us. @katie-lamb can probably provide more detail, but my understanding is that some subsidiary companies show up in exhibit 21's, but don't file a 10k and thus don't have a CIK (or at least not that we know of), so we assign this ID instead. I've updated the description to hopefully make this more clear.

Reply:
Okay, this sounds analogous to the `utility_id_ferc1` that we assign to provide a connection between the disparate `utility_id_ferc1_dbf` and `utility_id_ferc1_xbrl` IDs that come from FERC.  Unless we think this ID should apply to all instances of a given company across all SEC filings (which seems unlikely since we're assigning it in the context of this particular work) then I think it should have the more narrow `_sec10k` suffix. I think being explicit that we are the ones doing the assignment ("PUDL-assigned identifier...") as we do with the FERC1 IDs will help people know who to ask for help / blame if something is weird.